crates/lib/examples/local.rs

```diff
@@ -0,0 +1,29 @@
+use std::f64::consts::PI;
```

Comment:
did we want to rename/remove this before merging?

Reply:
Oh! I didn't realise that was an example that we added. Good spot.

Reply:
All the other examples have descriptive filenames. This one is just `local`. Maybe `with_parameter.rs`?